docs/index.rst

```diff
@@ -136,7 +136,7 @@ Need to contact us? Please use support@vertiq.co.
             :align: center
             
             Vertiq 60-XX Family
-     - .. figure:: ./_static/module_pictures/81xx/81xx_family.png
+     - .. figure:: ./_static/module_pictures/81xx/81xx_family_index.png
```

Comment:
I use a different image here vs. the family page. Here I use the same aspect ratio as the other, the family page I have less vertical dead space